packages/widget_toolkit/example/macos/Flutter/ephemeral/flutter_export_environment.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/opt/homebrew/Caskroom/flutter/3.0.0/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/stanislav.petkov/StudioProjects/widget_toolkit/packages/widget_toolkit/example"
+export "FLUTTER_ROOT=/Users/pwndp/fvm/versions/3.22.2"
```

Comment:
Do we really need to add user specific configurations such as /Users/pwndp ?

Reply:
Not sure why those files/paths were included with this PR, but I've listed them in gitignore.